include/RAJA/policy/openmp/reduce.hpp

```diff
@@ -15,6 +15,10 @@
 #define RAJA_omp_reduce_HPP
 
 #include "RAJA/config.hpp"
+#include "RAJA/pattern/detail/reduce.hpp"
```

Comment:
In light of our conversation yesterday about header files, would it be a good idea to pull all header inclusions inside the RAJA_ENABLE_OPENMP guard here? And similarly throughout the code?

Reply:
Fair point, doing it now.